src/Microsoft.Kusto.ServiceLayer/ObjectExplorer/Nodes/TreeNode.cs

```diff
@@ -51,7 +51,7 @@ protected TreeNode(IDataSource dataSource, DataSourceObjectMetadata objectMetada
         {
             DataSource = dataSource;
             ObjectMetadata = objectMetadata;
-            NodeValue = objectMetadata.Name;
+            NodeValue = objectMetadata.PrettyName;
```

Comment:
Changed the value of the node we render. Ie: these guys
![image](https://user-images.githubusercontent.com/63619224/97767718-34260900-1adb-11eb-9bc2-ce9ef6eae31a.png)